tests/parsing/grammar/expressions/expected/es6template.res.txt

```diff
@@ -33,46 +33,46 @@ let s =
       (({js|  after|js})[@res.template ]))
   [@res.template ])
 let s =
-  ((((((((({js||js})[@res.template ]) ^ foo)[@res.template ]) ^ (({js||js})
-         [@res.template ]))
-       [@res.template ]) ^ bar)
-      ^ (({js||js})[@res.template ]))
+  (((((((((({js||js})[@res.template ]) ^ foo)[@res.template ]) ^ (({js||js})
+          [@res.template ]))
+        [@res.template ]) ^ bar)
+      [@res.template ]) ^ (({js||js})[@res.template ]))
```

Comment:
There's an extra empty string in the AST now that I need to get rid of.

Reply:
The differences here are due to inconsistent use of `~attrs:[templateLiteralAttr]` in the old code.  The new code sets this attribute on all `Pexp_apply` nodes generated by `parseTemplateExpr`.

Reply:
Maybe I misunderstood thing, but isn't it normal to have a an empty string at the end? 
Like 
```
`${foo}`
``` 
should produce `['','']` for the template and `foo` for the sub expression.